rma_account/models/account_move.py

```diff
@@ -50,8 +50,6 @@ def _post_process_invoice_line_from_rma_line(self, new_line, rma_line):
             new_line._get_computed_name(),
```

Comment:
Replace new_line.name, since it becamse a computed field.

Reply:
Done, thanks @florian-dacosta!